zone/aggro.cpp

```diff
@@ -1278,6 +1295,39 @@ bool Mob::CheckLosFN(glm::vec3 posWatcher, float sizeWatcher, glm::vec3 posTarge
 	return zone->zonemap->CheckLoS(posWatcher, posTarget);
 }
 
+bool Mob::CheckPositioningLosFN(Mob* other, float x, float y, float z) {
```

Comment:
I'm not sure where all new this is being used but we should be cognizant that LOS calls are expensive